### DISCUSSION TITLE
Any future support for DuckDB Spatial?

### DISCUSSION THREAD

**Discussion by lawrencenika** [developmentseed/tipg#202] [Ideas]
DuckDB spatial has many similar functions to PostGIS too.

**Comment by vincentsarago** [comment]
We are still waiting for vector tile support to work on it

**Comment by bmcandr** [comment]
@lawrencenika I've been hacking on https://github.com/bmcandr/fast-geoparquet-features which uses `duckdb` to serve features + MVT/PBF from GeoParquet. It is heavily inspired by @vincentsarago's awesome work on tipg et al.
